onnx2tf/onnx2tf.py

```diff
@@ -65,7 +64,7 @@ def convert(
     copy_onnx_input_output_names_to_tflite: Optional[bool] = False,
     output_integer_quantized_tflite: Optional[bool] = False,
     quant_type: Optional[str] = 'per-channel',
-    quant_calib_input_op_name_np_data_path: Optional[List] = None,
+    custom_input_op_name_np_data_path: Optional[List] = None, # edit parameter
```

Comment:
As for the majority of the wording, there is no problem, so please erase the comments at the end. ` # edit parameter`